tests/ops/integration_tests/test_sql_task.py

```diff
@@ -776,6 +778,146 @@ async def test_mariadb_access_request_task(
     )
 
 
+@pytest.mark.integration
+@pytest.mark.integration_scylladb
+@pytest.mark.asyncio
+@pytest.mark.parametrize(
+    "dsr_version",
+    ["use_dsr_3_0", "use_dsr_2_0"],
+)
+class TestScyllaDSRs:
+    async def test_scylladb_access_request_task_no_keyspace(
+        self,
+        db,
+        policy,
+        integration_scylladb_config,
+        scylladb_integration_no_keyspace,
+        privacy_request,
+        dsr_version,
+        request,
+    ) -> None:
+        request.getfixturevalue(dsr_version)  # REQUIRED to test both DSR 3.0 and 2.0
+
+        with pytest.raises(ScyllaConnectorMissingKeyspace) as err:
```

Comment:
so this fails for DSR 3.0 with a `did not raise` error -- I think this is because the error is raised in a celery task and so the test doesn't catch it? I do see the exception in the logs. Not too sure how to fix this 😓